vowpalwabbit/gd.cc

```diff
@@ -1211,12 +1230,10 @@ base_learner* setup(VW::setup_base_i& stack_builder)
                .default_value(0.f)
                .help("Degree of l2 regularization applied to activated sparse parameters"))
       .add(make_option("l1_state", all.sd->gravity)
-               .keep(all.save_resume)
-               .default_value(0.)
+               .default_value(L1_STATE_DEFAULT)
```

Comment:
If the model file does not have l1,l2 (i.e. older model), and we don't pick up l1, l2 values from command line will this be breaking change?

Reply:
No, if neither the model file or command line contains l1/l2 information then the default value will be used as before. There is no change in behavior